src/main/java/com/easyrest/components/owner/Details.java

```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Details {
```

Comment:
as for me, better name is "RestaurantDetails"

Reply:
Done